.eslintrc.js

```diff
@@ -90,14 +90,19 @@ module.exports = {
           'index',
           'object',
         ],
-        'newlines-between': 'always',
         alphabetize: {
           order: 'asc',
           caseInsensitive: true,
         },
         warnOnUnassignedImports: false,
       },
     ],
+    'no-console': [
```

Comment:
There are log statements that I think are convenient to keep around for debugging (in dev). What do you think about dropping them at minification https://github.com/greenriver/hmis-frontend/pull/363 
I think it can co exist with this warning. It is helpful to see the warning in the editor to highlight any logs I may have unintentionally kept around.

Reply:
If for some reason we need info logging in prod, we could change it to allow `info`/`warn`/`error` in prod builds (filtering out `debug`/`log`).